.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: ruby
+dist: trusty
 
 rvm:
-  - 2.2
+  # - "2.2"
+  - "2.3.1"
 
-branches:
-  only:
-  - master
-  - develop
-  - deploy-release
-  - /^issue.*$/
+gemfile:
+  - cf.Gemfile
+
+# branches:
```

Comment:
I've also re-enabled the test matrix for ruby 2.2 as well as ruby 2.3, but I'm not entirely sure that ruby 2.2 is needed here since the cflinuxfs2 image is using 2.3 now.  The travis config was previously only testing on ruby 2.2.